### PR TITLE
feat: Add UnityVersion to telemetry

### PR DESCRIPTION
**Pull request by fuzzybinary**
### What and why?

Add the version of Unity used to create the Application to telemetry.  Also update the iOS SDK to the 2.8.0 release.

refs: RUM-3237

### Review checklist

- [ ] This pull request has appropriate unit and / or integration tests

### CODE REVIEW

File: samples/Datadog Sample/ProjectSettings/ProjectSettings.asset [hunk 1]
```diff
@@ -847,7 +847,7 @@ PlayerSettings:
   suppressCommonWarnings: 1
   allowUnsafeCode: 0
   useDeterministicCompilation: 1
-  additionalIl2CppArgs: '  --emit-source-mapping'
```
Comment (0xnm) [inline samples/Datadog Sample/ProjectSettings/ProjectSettings.asset:850]: is it related to the change?

File: samples/Datadog Sample/Build_BurstDebugInformation_DoNotShip/tempburstlibs/arm64-v8a/lib_burst_generated.txt
Comment (0xnm) [inline samples/Datadog Sample/Build_BurstDebugInformation_DoNotShip/tempburstlibs/arm64-v8a/lib_burst_generated.txt:1]: this file and file below probably shouldn't be committed?

File: samples/Datadog Sample/ProjectSettings/ProjectSettings.asset (hunk 1 above)
Comment (fuzzybinary) [reply samples/Datadog Sample/ProjectSettings/ProjectSettings.asset:850]: No but removing it is probably a good idea.

File: samples/Datadog Sample/Build_BurstDebugInformation_DoNotShip/tempburstlibs/arm64-v8a/lib_burst_generated.txt
Comment (fuzzybinary) [reply samples/Datadog Sample/Build_BurstDebugInformation_DoNotShip/tempburstlibs/arm64-v8a/lib_burst_generated.txt:1]: Yes, good catch.

**Review by 0xnm — APPROVED** [review]

**Pull request merged**